mdsa/algorithms/svd.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import scipy
+
+from mdsa.algorithm import Algorithm
+
+"""
+ Performs svd using a LAPACK (Linear Algebra Package) routine
+"""
+
+
+class Svd(Algorithm):
```

Comment:
Good question, and I'm not sure. Have you changed the name and run flake8 in your local machine to double check? Perhaps we can go based on that.